<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="box_red">
  	<static>false</static>
  	<link name="link">
      <inertial>
        <pose>0 0 0 0 0 0</pose>
        <!-- volume about 0.0085 m^3, density 2000 kg/m^3 -->
        <mass>5</mass>
        <!-- approximate as uniform box 0.37 x 0.17 x 0.20 m -->
        <inertia>
          <ixx>0.07</ixx>
          <ixy>0.0000</ixy>
          <iyy>0.07</iyy>
          <ixz>0.0000</ixz>
          <iyz>0.0000</iyz>
          <izz>0.07</izz>
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry>
          <box>
            <size>0.03 0.03 0.17</size>
          </box>
        </geometry>
        <surface>
          <contact>
            <ode>
              <max_vel>0.1</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
          <friction>
          	<ode>
          	  <mu>100</mu>
          	  <mu2>50</mu2>
          	</ode>
          </friction>
        </surface>
      </collision>
      <visual name="visual">
      	<geometry>
          <box>
            <size>0.03 0.03 0.17</size>
          </box>
        </geometry>
        <material>
          <script>
          	<uri>file://media/materials/scripts/gazebo.material</uri>
          	<name>Gazebo/Red</name>
          </script>
        </material>
      </visual>
    </link>
  </model>
</sdf>
